atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/Trace.common.kt

```diff
@@ -60,12 +60,24 @@ public expect fun TraceBase.named(name: String): TraceBase
 public expect val traceFormatDefault: TraceFormat
 
 public open class TraceBase internal constructor() {
-    @JsName("atomicfu\$Trace\$append\$")
+    @JsName("atomicfu\$Trace\$append\$1\$")
```

Comment:
Could you please elaborate on why `JsName` is required here?

Reply:
I use `JsName` to rely on this name while erasing these declarations from the js output